### PR TITLE
Add Components

### PR DESCRIPTION
**Pull request by tashisrael**
# In this PR, I implemented the following:
- Created a new branch for this part
- Deleted all the boilerplate from CRA (text, images, styles).
- Create a directory called `components`.
- Inside components, I created a new `Calculator.js` file.
- In `Calculator.js`, I created a React component that matches the given design

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,9 @@
...
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Calculator />
```
Comment (sja-thedude) [inline src/App.js]: - [x] Kindly make sure that the App.js component is class-based.
```
class App extends Component {
  
  } 
```

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        <button type="button">4</button>
+        <button type="button">5</button>
+        <button type="button">6</button>
+        <button type="button" className="sign">-</button>
+        <button type="button">1</button>
+        <button type="button">2</button>
+        <button type="button">3</button>
+        <button type="button" className="sign">+</button>
+        <button type="button" className="span_two">0</button>
+        <button type="button">.</button>
```
Comment (sja-thedude) [inline src/components/Calculator.js:29]: - [x] Kindly make sure to follow the design as per the requirements which is displayed on the left.

Screenshot 2022-10-26 at 7 56 05 PM

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @tashisrael, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ All linters passing
✔️ Correct gitflow
✔️ Professional Readme

## Required 
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (tashisrael) [reply src/App.js]: Thank you. Its now done

File: src/components/Calculator.js (hunk 2 above)
Comment (tashisrael) [reply src/components/Calculator.js:29]: Alright. Thanks

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @tashisrael 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

**Highlights**
- Well done on making the required changes
- Good communication

Cheers and Happy coding!👏👏👏

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**